src/pybind/mgr/cephadm/tests/test_service_discovery.py

```diff
@@ -123,7 +123,7 @@ def test_get_sd_config_node_exporter(self):
 
     def test_get_sd_config_alertmgr(self):
         mgr = FakeMgr()
-        root = Root(mgr)
+        root = Root(mgr, '', '', '', '')
```

Comment:
Then you don't need to change these, right?

Reply:
> Then you don't need to change these, right?

upsss I missed a couple of them. Fixed :+1: